src/components/common/TermsBanner/index.tsx

```diff
@@ -0,0 +1,51 @@
+import { AppRoutes } from '@/config/routes'
+import css from './styles.module.css'
+import { Button, Typography } from '@mui/material'
+import Link from 'next/link'
+import MUILink from '@mui/material/Link'
+import useLocalStorage from '@/services/local-storage/useLocalStorage'
+import { useEffect, useState } from 'react'
+
+const TERMS_KEY = 'terms_dismissed'
+
+const TermsBanner = () => {
```

Comment:
I would suggest we use the OnboardingTooltip for this. It implements all the same logic and looks more consistent with the rest of the theme.

Reply:
Currently the component only displays a small Tooltip when wrapped around a child but maybe we can extract some of it into a hook and reuse that.